include/cuco/static_map.cuh

```diff
@@ -296,6 +296,26 @@ class static_map {
             KeyEqual key_equal  = KeyEqual{},
             cudaStream_t stream = 0);
 
+  /**
+   * @brief Retrieves all of the keys and their associated values.
+   *
+   * The order in which keys are returned is implementation defined and not guaranteed to be
+   * consistent between subsequent calls to `retrieve_all`.
+   *
+   * Behavior is undefined if the range beginning at `keys_out` or `values_out` is not large enough
+   * to contain the number of keys in the map.
```

Comment:
```suggestion
   *
   * Behavior is undefined if the range beginning at `keys_out` or `values_out` is less than `size()`
```

Reply:
Are you referring to `get_size()`? Time to remove all `get_` prefixes.